hazelcast/src/main/java/com/hazelcast/spi/impl/waitnotifyservice/impl/WaitNotifyServiceImpl.java

```diff
@@ -86,8 +86,10 @@ public WaitNotifyServiceImpl(final NodeEngineImpl nodeEngine) {
     @Override
     public void populate(LiveOperations liveOperations) {
         for (Queue<WaitingOperation> queue : mapWaitingOps.values()) {
-            for (WaitingOperation op : queue) {
-                liveOperations.add(op.getCallerAddress(), op.getCallId());
+            for (WaitingOperation waitingOperation : queue) {
+                // we need to read out the data from the BlockedOperation; not from the ParkerOperation-container.
```

Comment:
a tiny one - the `ParkerOperation-container.` has a different name in the maintanance branch.